cmpirque/static_pages/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
This is not correct, the tests must be in a folder with that name, like `cmpirque/pages/tests/` It is done to modularize the tests by parts: `test_models`, `test_views`, etc.